lib/ipam/ipam.go

```diff
@@ -649,11 +697,12 @@ func (c ipamClient) autoAssign(ctx context.Context, num int, handleID *string, a
 					continue
 				}
 				logCtx.WithError(err).Warningf("Failed to assign IPs in newly allocated block")
+				ia.AddMsg("Failed to assign IPs in newly allocated block")
```

Comment:
Should we say which block we failed to assign IPs to?

Reply:
We talked offline and ended up deciding blocks are internal information to calico and there is not much gain in specifying them. We do specify which IP pools in the logging, though.